packages.json

```diff
@@ -23332,12 +23332,18 @@
       "iso",
       "countries",
       "country",
+      "language",
+      "languages",
+      "currency",
+      "currencies",
       "3166",
       "3166-1",
       "3166-2",
-      "3166-3"
+      "3166-3",
+      "15924",
+      "4217"
     ],
```

Comment:
You should be more explicit: `ISO-3166`, etc. for these tags.

Reply:
Updated with more explicit tags as per your request :)